### PR TITLE
org with com replaced

### PR DESCRIPTION
**Pull request by jnweiger**
All doc.owncloud.org replaced with useable doc.owncloud.com links. This also changes some tests. This is probably too much. Let's see...

### CODE REVIEW

File: tests/unit/Utils/DocLinkTest.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		$expected = 'https://doc.owncloud.com/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
```
Comment (pako81) [inline tests/unit/Utils/DocLinkTest.php:14]: Is this link expected to be working?

Comment (esgarov) [reply tests/unit/Utils/DocLinkTest.php]: it is fixed. please review again @pako81

**Review by pako81 — APPROVED** [review]

File: tests/unit/Utils/DocLinkTest.php [hunk 2]
```diff
@@ -11,9 +11,9 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		$expected = 'https://doc.owncloud.com/server/10.12/admin_manual/installation/installation_wizard.html#strong-perms-label';
```
Comment (mmattel) [inline tests/unit/Utils/DocLinkTest.php:14]:
```suggestion
		// This test constructs strings, from version and relative part and compares that
		// construct with the expected result. It does not check our actual servers. 
		$expected = 'https://doc.owncloud.com/server/10.12/admin_manual/installation/installation_wizard.html#strong-perms-label';
```
So we do not forget what this is about and version strings have no impact to reality.

File: tests/unit/Utils/DocLinkTest.php [hunk 3]
```diff
@@ -11,9 +11,11 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		// This test constructs strings, from version and relative part and compares that
+		// construct with the expected result. It does not check our actual servers.
```
Comment (jnweiger) [inline tests/unit/Utils/DocLinkTest.php:15]:
```suggestion
		// construct with the expected result. It does not check our actual servers.
```
no whitespace ant eol

**Review by mmattel — APPROVED** [review]

**Pull request merged**